### ISSUE TITLE
Is there any documentation of the stuff that was done with GRAKN?

### ISSUE THREAD

**Issue by DCGenomics** [NCBI-Hackathons/pheno_geno_ataxia#2]
If so, can you point me to it?

Thanks!

Ben

**Comment by oberstal** [comment]
No, we didn't end up using GRAKN during the hack--I got it running on my laptop easily enough but after trying 1000 different things I couldn't get it fully functioning on our vm, so it was a bust for team-use. I'm still investigating its functionality for my own research though if you just want to get a review of it...
